cliff.toml

```diff
@@ -24,6 +24,10 @@ commit_parsers = [
     { message = "^style", group = "Styling"},
     { message = "^chore\\(release\\): prepare for", skip = true},
     { message = "^chore", group = "Miscellaneous Tasks"},
+    { message = "^build", group = "Build system"},
+    { message = "^docs", group = "Documentation"},
+    { message = "^ci", group = "CI"},
+    { message = ".*", group = "Other"},
```

Comment:
This will also mean that we get all of dependabot's updates in the changelog at `Other`, is this intended?